### DISCUSSION TITLE
Добавление сторонних файлов в репозиторий

### DISCUSSION THREAD

**Discussion by Kukushechka7-ezx** [Sokol-Games-GmbH/PixelCombats.PublicApi#102] [Ideas]
Сделать возможность читать файлы из репозитория через апи, например:
`FileReader.Read('тут пишем имя или путь файла);`

А это запись в файл
`FileReader.Write('путь', 'текст для записи');`

**Comment by Kukushechka7-ezx** [comment]
Сотый)

**Comment by kkohno** [comment]
читать это сделаем, но вот запись - это уже вносит в режим изменения в процессе игры... по соображениям безопасности такое врядли будет, но можем сделать запись в базу данных, также можем сделать запись в файл на устройстве, чтобы сохранить какие-то значения на устройстве игрока, в процессе игры
